README.md

```diff
@@ -77,6 +77,9 @@ decimal by 1). Otherwise, the integration may complain of a duplicate unique ID.
 | `hide_temperatures`              | bool             | **Optional** | Whether to hide temperatures under the bar                                                                                                                      | `false`             |
 | `round_temperatures`             | bool             | **Optional** | Whether to round temperatures to the nearest whole number                                                                                                       | `false`             |
 | `hide_bar`                       | bool             | **Optional** | Whether to hide the bar itself                                                                                                                                  | `false`             |
+| `icon_fill`                      | bool             | **Optional** | Whether to repeat the icon inside the bar (one icon per segment)                                                                                                | `false`             |
+| `official_icons`                 | bool             | **Optional** | Whether to use the icons from the official Weather Forecast card                                                                                                | `false`             |
```

Comment:
I am ok with the `icon_fill` option. But I don't think that these alternate icons should go in. It adds unnecessary bloat to the package as most users won't use these other icons. I'd suggest using card-mod and CSS to hide the default icons and show your own in place using the CSS `content`rule.